protocols/identify/examples/identify.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright 2018 Parity Technologies (UK) Ltd.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a
+// copy of this software and associated documentation files (the "Software"),
+// to deal in the Software without restriction, including without limitation
+// the rights to use, copy, modify, merge, publish, distribute, sublicense,
+// and/or sell copies of the Software, and to permit persons to whom the
+// Software is furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+// OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+// FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+// DEALINGS IN THE SOFTWARE.
+
+//! identify example
+//!
+//! In the first terminal window, run:
+//!
+//! ```sh
+//! cargo run --example identify
+//! ```
+//! It will print the [`PeerId`] and the listening addresses, e.g. `Listening on
+//! "/ip6/2001:db8:: /tcp/24915"`
+//!
+//! In the second terminal window, start a new instance of the example with:
+//!
+//! ```sh
+//! cargo run --example identify -- /ip4/127.0.0.1/tcp/24915
+//! ```
+//! The two nodes establish a connection, negotiate the identify protocol
+//! The dialing node prints out the `PeerId` of the node it is sending identify info to
+//! The other node prints out the received identify info.
```

Comment:
I think not only the dialer, but instead both peers will sent each other identify info once they establish a connection, no?